packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -665,10 +706,34 @@ export const Webchat = forwardRef((props, ref) => {
                 Object.keys(webchatState.replies).length > 0 &&
                 webchatReplies()}
               {persistentMenuIsOpened && (
-                <OpenedPersistentMenu
-                  onClick={closeMenu}
-                  options={persistentMenuOptions}
-                />
+                <div>
+                  {darkBackgroundMenu && (
+                    <div
+                      onClick={closeMenu}
+                      style={{
+                        backgroundColor: '#000000',
+                        position: 'absolute',
+                        width: '100%',
+                        height: '100%',
+                        opacity: 0.3,
+                        zIndex: 1,
+                        right: 0,
+                        bottom: 0,
+                      }}
+                    />
```

Comment:
please, convert it into a styled-component and try to use the colors defined in constants